src/DataCollection.UWP/LocalizedStrings/Resources.resw

```diff
@@ -540,4 +540,34 @@
   <data name="AddFeatureError_Title" xml:space="preserve">
     <value>Error adding feature</value>
   </data>
+  <data name="DTE_DateButton_Tooltip" xml:space="preserve">
+    <value>Select a date</value>
+  </data>
+  <data name="DTE_DatePart_Placeholder" xml:space="preserve">
+    <value>mm/dd/yyyy</value>
+  </data>
+  <data name="DTE_ResetButton_Tooltip" xml:space="preserve">
+    <value>Clear date and time</value>
+  </data>
+  <data name="DTE_TimeButton_Tooltip" xml:space="preserve">
+    <value>Select a time</value>
+  </data>
+  <data name="DTE_TimePart_Placeholder" xml:space="preserve">
+    <value>hh:mm AM/PM</value>
```

Comment:
If the user enters a time in 24-hour format or omits AM/PM, it will be converted to 12 hour AM/PM format. AM/PM isn't necessarily required, but it is clear about the behavior and expectations. Lowercase am/pm also works, but will be shown as capital AM/PM once committed

Reply:
As OSA needs it, I think this solution is good as-is. I think it would be worth considering if the Calcite team would want locale support baked in to the tool in its initial implementation.